src/component/ActionMappingForm.tsx

```
@@ -1,225 +1,377 @@
+                onChange={(e) => {
+                    changeEditTarget(e.target.value);
+                    console.log("setPresetTmp", presetTmp);
+                }}
+                value={editTarget}
+            >
+                <option selected value="">
+                    [new preset]
+                </option>
+                {mappingList.current.map((a) => {
+                    return (
+                        <option value={a[0]}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)